src/query/storages/iceberg/src/database.rs

```diff
@@ -116,4 +125,112 @@ impl Database for IcebergDatabase {
             .map(|table_name| table_name.name.to_string())
             .collect())
     }
+
+    // create iceberg_catalog.db.t (col_name type);
+    #[async_backtrace::framed]
+    async fn create_table(&self, req: CreateTableReq) -> Result<CreateTableReply> {
+        let table_name = TableIdent::new(self.ident.clone(), req.table_name().to_string());
+
+        match req.create_option {
+            CreateOption::Create => {}
+            CreateOption::CreateIfNotExists => {
+                if self
+                    .ctl
+                    .iceberg_catalog()
+                    .table_exists(&table_name)
+                    .await
+                    .map_err(|err| {
+                        ErrorCode::Internal(format!(
+                            "Iceberg create table {}.{} failed: {err:?}",
+                            req.db_name(),
+                            req.table_name()
+                        ))
+                    })?
+                {
+                    return Ok(CreateTableReply {
+                        table_id: 0,
+                        table_id_seq: None,
+                        db_id: 0,
+                        new_table: true,
+                        spec_vec: None,
+                        prev_table_id: None,
+                        orphan_table_name: None,
+                    });
+                }
+            }
+            CreateOption::CreateOrReplace => {
+                self.drop_table_by_id(DropTableByIdReq {
+                    if_exists: true,
+                    tenant: req.tenant().clone(),
+                    tb_id: 0,
+                    table_name: req.table_name().to_string(),
+                    db_id: 0,
+                    db_name: req.db_name().to_string(),
+                    engine: req.table_meta.engine.to_string(),
+                    session_id: "".to_string(),
+                })
+                .await?;
+            }
+        }
+
+        let arrow_schema: arrow_schema::Schema = req.table_meta.schema.as_ref().into();
```

Comment:
Let's add another function to inject the metadata.

The bloom filter tests should not be changed.